### DISCUSSION TITLE
Beginner pushing to github in browser from desktop

### DISCUSSION THREAD

**Discussion by ukudaly** [community/community#23802] [New to GitHub]
<p>I used Github Desktop to create a repository and it shows in the browser.</p>
<p>When I’ve moved files into the respository on my desktop and they show in the desktop app. I then commited these files to the Master branch. Now when I try to publish the branch it tells me Authentification failed.</p>
<p>Sorry this is probably something super simple but I’m brand new to this!</p>
<p>Thanks.</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Here’s our guide to authenticating to GitHub from GitHub Desktop:</p>
<p><a href="https://help.github.com/en/desktop/getting-started-with-github-desktop/authenticating-to-github" class="onebox" target="_blank" rel="nofollow noopener">https://help.github.com/en/desktop/getting-started-with-github-desktop/authenticating-to-github</a></p>
<p>Or alternatively, you can get GitHub Desktop to authenticate by using your login in the browser:</p>
<p><a href="https://help.github.com/en/desktop/getting-started-with-github-desktop/authenticating-to-github-using-the-browser" class="onebox" target="_blank" rel="nofollow noopener">https://help.github.com/en/desktop/getting-started-with-github-desktop/authenticating-to-github-using-the-browser</a></p>
